.gitignore

```diff
@@ -101,3 +101,6 @@ aux_*.sh
 node_modules
 include
 kytos/web-ui/source/static/css/*
+
+# ignore templates
+*.conf
```

Comment:
You are right, but when installing in "development mode" (`python setup.py develop`) the installer should create the `kytos.conf` file, put it on the `etc/kytos` directory (on the repository dir) and do the symbolic link to this directory. So, we will have a `kytos.conf` file under the git tree. Therefore adding it to gitignore seems to be important to avoid commiting it by mistake. =)